src/Core/src/ImageSources/FileImageSourceService/FileImageSourceService.iOS.cs

```diff
@@ -41,7 +41,7 @@ public partial class FileImageSourceService
 			{
 				Logger?.LogWarning(ex, "Unable to load image file '{File}'.", imageSource.File);
 
-				throw;
+				return FromResult(null);
```

Comment:
Based on https://github.com/dotnet/maui/pull/27175/files#r1939317042
This align the behavior between platforms. LGTM.